test/promql/tests/data_plane/latency_image_repository_provision_test.yaml

```diff
@@ -0,0 +1,101 @@
+evaluation_interval: 1m
+
+rule_files:
+  - 'prometheus.latency_image_repository_provision_alerts.yaml'
+
+tests:
+# Scenario where only one cluster cross the 5% threshold
```

Comment:
Better to align the the comments with the block it relates to, as you did for the `input_series`. I suspect `yamllint` will also complain of this comment and the similar ones below.

Reply:
linter error was because I had in other yaml "#TODO" text